public/app/plugins/datasource/cloudwatch/components/QueryEditor/LogsQueryEditor/LogsQueryEditor.tsx

```diff
@@ -1,16 +1,14 @@
 import { css } from '@emotion/css';
 import React, { memo } from 'react';
 
-import { AbsoluteTimeRange, QueryEditorProps } from '@grafana/data';
-import { config } from '@grafana/runtime';
+import { QueryEditorProps } from '@grafana/data';
 import { InlineFormLabel } from '@grafana/ui';
 
 import { CloudWatchDatasource } from '../../../datasource';
 import { CloudWatchJsonData, CloudWatchLogsQuery, CloudWatchQuery } from '../../../types';
 
 import { CloudWatchLink } from './CloudWatchLink';
 import CloudWatchLogsQueryFieldMonaco from './LogsQueryField';
```

Comment:
```suggestion
import CloudWatchLogsQueryField from './LogsQueryField';
```

Minor nit: let's remove `Monaco` for the name since it was only used to differentiate from the previous component.